recipes/dask-searchcv/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "dask-searchcv" %}
+{% set version = "0.0.1" %}
+{% set md5 = "7d8fb38ba01d7518e2d22e5933f1e419" %}
```

Comment:
Ah. That seems to be true only for the pre-production warehouse deployment. Current pypi doesn't: https://pypi.python.org/pypi/dask-searchcv/0.0.1. Fixed to use sha256.